go/test/apis/devfile_recognizer_test.go

```diff
@@ -170,7 +170,7 @@ func getDevFileTypes() []model.DevFileType {
 			ProjectType: "spring",
 			Tags: []string{
 				"Java",
-				"Spring",
```

Comment:
The original mock test is to test tag `Spring`. is that possible to add another test for `Spring Boot` instead of removing the coverage for `Spring`?